sdk/translation/ai-translation-text-rest/test/public/translateTest.spec.ts

```diff
@@ -266,10 +262,12 @@ describe("Translate tests", () => {
 
   it("with profanity", async () => {
     const inputText: InputTextItem[] = [{ text: "shit this is fucking crazy" }];
-    const parameters: TranslateQueryParamProperties & Record<string, unknown> = {
+    const profanityAction: ProfanityAction = "Marked";
```

Comment:
why not remove the type annotation here?

Reply:
I am getting following error:
```
test/public/translateTest.spec.ts:272:7 - error TS2322: Type '{ to: string; profanityAction: string; profanityMarker: string; }' is not assignable to type 'TranslateQueryParamProperties & Record<string, unknown>'.
  Type '{ to: string; profanityAction: string; profanityMarker: string; }' is not assignable to type 'TranslateQueryParamProperties'.
    Types of property 'profanityAction' are incompatible.
      Type 'string' is not assignable to type 'ProfanityAction | undefined'.

272       queryParameters: parameters,
          ~~~~~~~~~~~~~~~

  src/generated/parameters.ts:138:3
    138   queryParameters: TranslateQueryParamProperties;
          ~~~~~~~~~~~~~~~
    The expected type comes from property 'queryParameters' which is declared here on type 'TranslateParameters'


Found 1 error in test/public/translateTest.spec.ts:272
```

Reply:
Could you try something like the below ?
```
     queryParameters: {
        to: "zh-cn",
        profanityAction: "Marked",
        profanityMarker: "Asterisk"
      },
    });
```
I think it should work, we always recommend inline objects for RLC.
Also, could you remove other type annotations I notice there's still something left like InputTextItem[] and it should be safe to remove.